### PR TITLE
ready: thats all for today cell

### PR DESCRIPTION
**Pull request by akhrail1996**
https://user-images.githubusercontent.com/16062322/125663139-d0a9f772-b99c-42f6-818b-2775c779e802.MOV

### CODE REVIEW

File: another/Feed/FeedCoordinator.swift [hunk 1]
```diff
@@ -109,11 +109,11 @@ final class FeedCoordinator: BaseCoordinator {
...
         case .match:
-            sheet.addAction(.init(title: "Unmatch", style: .destructive) { [unowned self] action in unmatch(profile) })
-            sheet.addAction(.init(title: "Report user", style: .destructive) { [unowned self] action in report(profile) })
-        case .profileEdit: ()
+            sheet.addAction(.init(title: "Unmatch", style: .destructive) { [unowned self] action in unmatch(profile!) })
+            sheet.addAction(.init(title: "Report user", style: .destructive) { [unowned self] action in report(profile!) })
+        case .profileEdit, nil: ()
         default:
-            sheet.addAction(.init(title: "Report user", style: .destructive) { action in print("reported") })
+            sheet.addAction(.init(title: "Report user", style: .destructive) { [unowned self] action in report(profile!) })
```
Comment (akhrail1996) [inline another/Feed/FeedCoordinator.swift:116]: @ruslandoga that was my bad, really sorry :(

File: another/Feed/FeedCoordinator.swift [hunk 2]
```diff
@@ -71,9 +71,9 @@ final class FeedCoordinator: BaseCoordinator {
         router.setRoot(to: feedVC, animated: true)
     }
     
-    private func showSheet(_ item: FeedItem) {
-        let profile = item.profile
-        let kind = item.kind
+    private func showSheet(_ item: FeedItem?) {
+        let profile = item?.profile ?? nil
+        let kind = item?.kind ?? nil
```
Comment (ruslandoga) [inline another/Feed/FeedCoordinator.swift:76]: `let kind = item?.kind` should be enough

Comment (ruslandoga) [reply another/Feed/FeedCoordinator.swift:76]: also, it shouldn't be nil

Comment (akhrail1996) [reply another/Feed/FeedCoordinator.swift:76]: @ruslandoga how would you presentSheet from "That's it for today" cell then?

One way is to create a separate broadcast

**Review by ruslandoga — APPROVED** [review]

File: another/Feed/FeedCoordinator.swift (hunk 2 above)
Comment (ruslandoga) [reply another/Feed/FeedCoordinator.swift:76]: I'd extend feed item to include that's id for today

Comment (ruslandoga) [reply another/Feed/FeedCoordinator.swift:76]: since that cell is still part of feed

Comment (akhrail1996) [reply another/Feed/FeedCoordinator.swift:76]: @ruslandoga then make feedProfile optional?

Comment (ruslandoga) [reply another/Feed/FeedCoordinator.swift:76]: no, feeditem can be enum

Comment (akhrail1996) [reply another/Feed/FeedCoordinator.swift:76]: @ruslandoga agreed, I will record this task for later

Comment (akhrail1996) [reply another/Feed/FeedCoordinator.swift:76]: since it turned out quite a lot need to be changed

**Pull request merged**